src/views/jobManager/jobManagerView.ts

```diff
@@ -0,0 +1,147 @@
+import vscode, { MarkdownString, ProgressLocation, ThemeIcon, TreeItem, TreeItemCollapsibleState, commands, env, window, workspace } from "vscode";
+import { TreeDataProvider } from "vscode";
+import { Config, JobManager } from "../../config";
+import { JobInfo } from "../../connection/manager";
+import { editJobUi } from "./editJob";
+import { displayJobLog } from "./jobLog";
+
+const selectJobCommand = `vscode-db2i.jobManager.selectJob`;
+const activeColor = new vscode.ThemeColor(`minimapGutter.addedBackground`);
+
+export class JobManagerView implements TreeDataProvider<any> {
+  private _onDidChangeTreeData: vscode.EventEmitter<vscode.TreeItem | undefined | null | void> = new vscode.EventEmitter<vscode.TreeItem | undefined | null | void>();
+  readonly onDidChangeTreeData: vscode.Event<vscode.TreeItem | undefined | null | void> = this._onDidChangeTreeData.event;
+
+  constructor(context: vscode.ExtensionContext) {
+    context.subscriptions.push(
+      vscode.commands.registerCommand(`vscode-db2i.jobManager.refresh`, async () => {
+        this.refresh();
+      }),
+
+      vscode.commands.registerCommand(`vscode-db2i.jobManager.newJob`, async () => {
+
+        await window.withProgress({location: ProgressLocation.Window}, async (progress) => {
+          try {
+            progress.report({message: `Spinning up SQL job..`});
```

Comment:
```suggestion
            progress.report({message: `Spinning up SQL job...`});
```
Should we substitute `Starting` for `Spinning up`?

Reply:
I like spinning up. Has a nice personal feeling to it.